Libraries/HyperV.psm1

```diff
@@ -348,11 +348,22 @@ function Create-HyperVGroupDeployment([string]$HyperVGroupName, $HyperVGroupXML,
             $newVhdName = "{0}-{1}{2}" -f @($vhdName, $infoParentOsVHD.DiskIdentifier.Replace("-", ""),$vhdSuffix)
             $localVHDPath = "{0}{1}{2}" -f @($hypervVHDLocalPath,[System.IO.Path]::DirectorySeparatorChar,$newVhdName)
             $localVHDUncPath = $localVHDPath -replace '^(.):', "\\${HyperVHost}\`$1$"
+            $CopyVHD = $false
             if ((Test-Path $localVHDUncPath)) {
                 Write-LogInfo "${parentOsVHDPath} is already found at path ${localVHDUncPath}"
+                $TargetVHD = Get-ChildItem -Path $localVHDUncPath
```

Comment:
here it should be Get-Item, why are you using child?

Reply:
@ader1990 , no specific reason there. Get-Item / Get-ChildItem works same for the absolute filepath.